tests/gorepo_test.go

```diff
@@ -22,3 +25,21 @@ func TestGoRepositoryCompilerTests(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// Test that GopherJS can be vendored into a project, and then used to build Go programs.
+// See issue https://github.com/gopherjs/gopherjs/issues/415.
+func TestGopherJSCanBeVendored(t *testing.T) {
+	if runtime.GOARCH == "js" {
+		t.Skip("test meant to be run using normal Go compiler (needs os/exec)")
+	}
+
+	cmd := exec.Command("bash", "gopherjsvendored_test.sh")
+	cmd.Stderr = os.Stdout
```

Comment:
Why not catching the stderr and dumping it as an error message?

Reply:
The test is considered failed if the exit code is non-zero, or if the stdout output is not exactly the expected "hello using js pkg\n" value.

Stderr can be anything, it shouldn't fail the test. It's just information.

For example, if you run the test without the sourcemap module, it'll print "gopherjs: Source maps disabled. Install source-map-support module for ..." message to stderr.

Reply:
Makes sense.